chore: Migrate @packages/electron to TS (#32417)
cypress-io/cypress: packages/electron/BUILD.md

# Electron Package Build Setup

> This package is configured to build both CommonJS and ES Module versions using TypeScript compiler (tsc) only.

Author: ```suggestion
This package is configured to build to CJS, but ESM builds can easily be enabled.
```

## Build Commands

```bash

> # Build both CJS and ESM versions

Author: ```suggestion
# Build CJS - ESM is disabled for now
```

npm run build

# Build only CommonJS
npm run build:cjs

# Build only ES Modules  
npm run build:esm

# Clean build output
npm run clean
```

## Output Structure

> After building, the `dist/` directory contains:

Author: ```suggestion
After building both `build:esm` and `build:cjs`, the `dist/` directory contains:
```
